### PR TITLE
docs: typedoc generation, update of contributing team

### PR DESCRIPTION
**Pull request by TheReincarnator**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,109 +1,143 @@
...
+* [Lasse Küchler](http://github.com/LKuechler)
+* [Luca Oelsner](http://github.com/lucoel)
+* [Mario Nebl](http://github.com/marionebl)
+* [Markus Ölhafen](http://github.com/markusoelhafen)
+* [Thomas Jacob](http://github.com/TheReincarnator)
+* [Tilman Frick](http://github.com/tilmx)
 
 ## Questions?
+
 Just drop us an [email](alva@sinnerschrader.com) or write a direct message on Twitter to [@meetalva](https://twitter.com/meetalva)
```
Comment (tilmx) [inline CONTRIBUTING.md]: Can we change the mail address to hey@meetalva.io?

Comment (TheReincarnator) [reply CONTRIBUTING.md]: Sure.

Comment (TheReincarnator) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,109 +1,143 @@
...
+## Commit message conventions
 
-### Commit message conventions
 For commit message we use the [angular commit guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
-#### commitizen & commitlint
+
+### commitizen & commitlint
+
 https://github.com/marionebl/commitlint
 https://github.com/commitizen/cz-cli
```
Comment (tilmx) [inline CONTRIBUTING.md]: The links are displayed next to each other and not below. I would say it looks more readable when they are below each other.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,109 +1,143 @@
...
 
+* bugs / feature requests: https://github.com/meetalva/alva/issues
+* website: https://meetalva.io
+* sprint board: https://github.com/meetalva/alva/projects/2
+* backlog board: https://github.com/meetalva/alva/projects/3
 
 ## Setup for contributors
+
 To add a contribution to Alva (improve its features, fix issues),
 get the source of the application itself, and if you don´t already have a Alva compatible styleguide, the kickstart styleguide (designkit):
```
Comment (tilmx) [inline CONTRIBUTING.md]: False apostrophe. Should be " ’ ".

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,109 +1,143 @@
...
 2. Run in the main (Alva) repository:
+
 ```shell
 npm i && npm start
 ```
 
 3. Now open a compatible styleguide.
 
-If you don´t have any, follow the next steps
+If you don´t have any, follow the next steps:
```
Comment (tilmx) [inline CONTRIBUTING.md]: False apostrophe. Should be " ’ ".

File: CONTRIBUTING.md (hunk 3 above)
Comment (TheReincarnator) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (TheReincarnator) [reply CONTRIBUTING.md]: Done. Converted into a readable sentence.

File: CONTRIBUTING.md (hunk 4 above)
Comment (TheReincarnator) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,109 +1,140 @@
...
 2. Run in the main (Alva) repository:
+
 ```shell
 npm i && npm start
 ```
 
 3. Now open a compatible styleguide.
 
-If you don´t have any, follow the next steps
+If you don't have any, follow the next steps:
```
Comment (tilmx) [inline CONTRIBUTING.md]: Maybe we should specify: "If you don’t have any compatible styleguide [...]" and add a small space above the section?

Comment (TheReincarnator) [reply CONTRIBUTING.md]: Done.

**Pull request merged**